storage/google/cloud/storage/blob.py

```diff
@@ -319,6 +319,9 @@ def download_to_file(self, file_obj, client=None):
         :param client: Optional. The client to use.  If not passed, falls back
                        to the ``client`` stored on the blob's bucket.
 
+        :type range_bytes: tuple
+        :param range_bytes: Optional. Range of bytes to download.
```

Comment:
do you want me to change all Optional->(Optional) ?  I was following the style of other optional parameters ?